cunumeric/deferred.py

```diff
@@ -606,6 +606,20 @@ def _broadcast(self, shape):
 
         return result
 
+    def _convert_future_to_store(self, a):
+        store = self.context.create_store(
+            a.dtype,
+            shape=(1,),
```

Comment:
Is there a reason you don't pass `a.shape` here? I suspect doing this could potentially require broadcasting later if the shape was not `(1,)` but some other degenerate case (e.g., `(1,1,)`).

Reply:
I have assumed that shape for the Future is always (1,). I will change it to `a.shape`